echopype/echodata/combine.py

```diff
@@ -54,13 +63,30 @@ def check_zarr_path(zarr_path: str, storage_options: Optional[dict]) -> str:
             source_file = path_obj.parts[-1]
             save_path = path_obj.parent
 
-    return validate_output_path(
+    validated_path = validate_output_path(
```

Comment:
Currently this validation will fail if I pass in path like `s3://zarr-bucket-test/combined.zarr`, this is because by the time save_path is passed into `validate_output_path` it becomes `s3:/zarr-bucket-test/` since you turned `zarr_path` into a Path object. You should just pass on `zarr_path` and let the `validate_output_path` do the work like so:

```python
    source_file = "combined_echodatas.zarr"
    if not zarr_path.strip("/").endswith(".zarr"):
        raise ValueError("The provided zarr_path input must have '.zarr' suffix!")

    validated_path = validate_output_path(
        source_file=source_file,
        engine="zarr",
        output_storage_options=storage_options,
        save_path=zarr_path,
    )
```